recipes/mss/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "1.2.1" %}
+
+package:
+    name: mss
+    version: {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: c08e72950a1150fd2a119a7fe766f966fe5a8b73c55fa02763c7a1e5a288f915
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+    skip: True  # [py3k]
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        # Relaxed versions of the run dependencies for conda-forge.
+        - python
+        - basemap-data-hires
```

Comment:
@ReimarBauer what do you want to do about `basemap` on conda-forge? Latest version or pin to `1.0.7`?

PS: Yesterday we had a BoF here in SciPy and everybody strongly discouraged the use of `basemap`. It is in maintenance mode now and the devs are recommending `cartopy`.